index.js

```diff
@@ -0,0 +1,81 @@
+import * as Navigation from './modules/nav-menu.js';
+import Book from './modules/Book-storage.js';
+import { DateTime } from './modules/luxon.js';
+
+Navigation.nav();
+/* eslint-disable */
+import  store  from './modules/display.js';
+/* eslint-enable */
```

Comment:
- [ ]  As required by the previous reviewer it's better to fix linters and not disable it. Please do not disable your linters, fixing linters helps to write a clear and good codebase 🙃 🙃